### PR TITLE
Add all data for the ratios topic

### PR DESCRIPTION
**Pull request by vinitamurthi**
## Explanation
This PR adds all the data pertaining to the ratios and proportional reasoning topic.

## Checklist

- [ ] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [ ] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [ ] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [ ] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [ ] The PR is made from a branch that's **not** called "develop".
- [ ] The PR is **assigned** to an appropriate reviewer.

### CODE REVIEW

**Comment by vinitamurthi** [comment]
@BenHenning @jamesxu0 PTAL

**Review by jamesxu0 — APPROVED** [review]
LGTM

**Comment by vinitamurthi** [comment]
@BenHenning PTAL!

File: domain/src/test/java/org/oppia/domain/exploration/ExplorationDataControllerTest.kt [hunk 1]
```diff
@@ -160,8 +164,9 @@ class ExplorationDataControllerTest {
     val explorationLiveData = explorationDataController.getExplorationById(FRACTIONS_EXPLORATION_ID_0)
     advanceUntilIdle()
     explorationLiveData.observeForever(mockExplorationObserver)
-
     verify(mockExplorationObserver, atLeastOnce()).onChanged(explorationResultCaptor.capture())
+    print(explorationResultCaptor.value.getErrorOrNull())
```
Comment (BenHenning) [inline domain/src/test/java/org/oppia/domain/exploration/ExplorationDataControllerTest.kt]: Debug line? Please remove.

**Review by BenHenning — APPROVED** [review]
Thanks @vinitamurthi!

File: domain/src/test/java/org/oppia/domain/exploration/ExplorationDataControllerTest.kt (hunk 1 above)
Comment (vinitamurthi) [reply domain/src/test/java/org/oppia/domain/exploration/ExplorationDataControllerTest.kt]: Done

**Pull request merged**